### ISSUE TITLE
Randomly-generated Blueprint Identifier Causes Frequent [ModuleName].xcscheme Conflicts

### ISSUE THREAD

**Issue by paulrehkugler** [CocoaPods/CocoaPods#5057]
:warning: A very similar issue (https://github.com/CocoaPods/CocoaPods/issues/1786) was closed as a duplicate of an issue that was resolved in 0.38.0.beta.2 (https://github.com/CocoaPods/CocoaPods/issues/1118). :warning: 

`BlueprintIdentifier` in the `[ModuleName].xcscheme` file is changed at every `pod install`. This is one of the more frequent causes of git conflicts, and a common solution is to just run `pod install` again.

Is this identifier truly random? Is it okay to manually resolve this conflict, instead of running `pod install` to fix it? Manually editing those files is probably quicker than waiting for a `pod install`.

Or, even better, is there a setting I can set to reuse the previous `BlueprintIdentifier` to just avoid these conflicts entirely?

**Comment by orta** [comment]
I thought deterministic UUIDs stopped this, do you have it disabled?

**Comment by paulrehkugler** [comment]
@orta Thanks for your response.

I don't believe we have that disabled. How would I check that to make sure?

Disabling deterministic UUIDs would cause changes in the `Pods/project.pbxproj` file for every `pod install`, right? If there are no changes to our dependencies or the Podfile, the only changes that show up in git due to a `pod install` are the `BlueprintIdentifier`s in the `*.xcscheme` files.

**Comment by mrackwitz** [comment]
@paulrehkugler: You can disable it via your Podfile or your `~/.cocoapods/config.yaml`, but from what you describe it doesn't seem like that would be the issue.

**Comment by mrackwitz** [comment]
The `BlueprintIdentifier` is the `UUID` of the referenced target of your project. Those are deterministically generated. I think this issue was fixed by #4811, which generates the deterministic IDs first and then recreates the schemes with the updated IDs.
@paulrehkugler: Which version of CocoaPods are you using?

**Comment by paulrehkugler** [comment]
We are using version `0.39.0`:

```
Paul-Rehkuglers-MacBook-Pro:orangina paul$ bundle exec pod --version
0.39.0
```

**Comment by segiddins** [comment]
Please check if this is fixed in 1.0, as https://github.com/CocoaPods/CocoaPods/pull/4811 was not included in 0.39.

**Comment by segiddins** [comment]
Closing due to lack of response.
